src/components/views/rooms/NotificationBadge/StatelessNotificationBadge.tsx

```diff
@@ -28,7 +28,12 @@ interface Props {
     count: number;
     level: NotificationLevel;
     knocked?: boolean;
-    type?: "badge" | "dot";
+    /**
+     * If true, where we would normally show a badge, we instead show a dot. Count will
```

Comment:
Do we mean `count` the property here, or "Notification count" as a concept? Given the text about it affecting whether the dot is displayed, I think we mean `count`? In which case, backticks and lower case.

Reply:
Mmm... I would say more the notification count concept: I'm not clear what exactly it would mean to display or not display a prop. I've referenced the count in the class doc and updated this to hopefully be clearer in 940e9d34ac.